src/index.html

```diff
@@ -1,17 +1,184 @@
+            name="surname"
+            id="surname"
+            required
+            autocomplete="off">
+        </div>
+
+
+        <!-- /* name */ -->
+        <div class="input">
+          <label for="name">Name:</label>
+          <input type="text"
+            name="name"
+            id="name"
+            required
+            autocomplete="off">
+        </div>
+
+
+        <!-- /* age */ -->
+        <div class="input">
+          <label for="age">How old are You?</label>
+          <input type="number"
+            name="age"
+            id="age"
+            required
+            min="1"
+            max="100"
+            value="12">
+        </div>
+
+
+        <!-- /* birthday */ -->
+        <div class="input">
+          <label for="birth">Full date of birth:</label>
+          <input type="date"
+            name="birth"
+            id="birth"
+            required>
+        </div>
+
+        <!-- /* accept */ -->
+        <div class="input">
+          <label for="accept">I accept the term of the agreement</label>
+          <input type="checkbox"
+            name="accept"
+            id="accept">
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <!-- /* email */ -->
+        <div class="input">
+          <label for="email">E-mail:</label>
+          <input type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required>
+        </div>
+
+        <!-- /* email */ -->
+        <div class="input">
+          <label for="password">Password:</label>
+          <input type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="25"
+            required>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <!-- /* love cats? */ -->
+        <div class="input">
+          Do you love cats?
+          <label>
+            <input type="radio"
+              name="love_cats"
+              value="true"> Yes
+          </label>
+          <label>
+            <input type="radio"
+              name="love_cats"
+              value="false"> No
+          </label>
+        </div>
+
+        <!-- /* color */ -->
+        <div class="input">
+          <label for="color">What is your favorite color?</label>
+          <input type="color"
+            name="color"
+            id="color"
+            required>
+        </div>
+
+        <!-- /* time */ -->
+        <div class="input">
+          <label for="time">What time do you go to bed?</label>
+          <input type="time"
+            name="time"
+            id="time"
+            required>
+        </div>
+
+        <!-- /* select */ -->
+        <div class="input">
+          <label for="select">What are your favorite brand of cars?</label>
+          <select name="select"
```

Comment:
add more specific name

```suggestion
          <select name="carBrand"
```